stack_definition.json

```diff
@@ -315,7 +365,7 @@
       ]
     },
     {
-      "name": "5 - Sample RAG app - Application Lifecycle Management",
+      "name": "6 - Sample RAG app - Application Lifecycle Management",
```

Comment:
@akocbek I think your going to have to do a manual upgrade test to see how projects handles the renaming of a config. I suspect in may not rename on upgrade, but we need to test

Reply:
Should we just add elastic as "5a" after Watson SaaS?